README.md

```diff
@@ -2,6 +2,17 @@
 
 When this library is added to a project you may configure a Java Mail session to use the `smtp+smime` or `smtps+smime` protocol to have the transport sign outgoing mail with a mail certificate stored in a Java keystore (JKS) file.
 
+## Installation
+### From Maven
+Add the following as a dependency in your project
+```xml
+<dependency>
+    <groupId>edu.iu.uits</groupId>
+    <artifactId>javamail-smime-transport</artifactId>
+    <version>1.0.10</version>
```

Comment:
Including the most recent version in the installation instructions seems to be common practice, so it's something we'll have to remember as part of the release process.